client/blocks/taxonomy-manager/index.jsx

```diff
@@ -11,27 +12,66 @@ import SectionHeader from 'components/section-header';
 import Card from 'components/card';
 import SearchCard from 'components/search-card';
 import Button from 'components/button';
+import TermsList from './list';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getPostTypeTaxonomy } from 'state/post-types/taxonomies/selectors';
 
 export class TaxonomyManager extends Component {
 	static propTypes = {
-		translate: PropTypes.func,
 		taxonomy: PropTypes.string,
+		labels: PropTypes.object,
+		postType: PropTypes.string,
 	};
 
+	static defaultProps = {
+		postType: 'post'
+	};
+
+	constructor() {
+		super( ...arguments );
```

Comment:
I usually use `super(props)`, is this some kind of a best practice, may be, to avoid knowing about the args if they get changed ?